docs/conf.py

```diff
@@ -10,100 +10,211 @@
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
+# -- Project information -----------------------------------------------------
+
 import os
 import sys
-import datetime
-# import sphinx_book_theme
+import pathlib
+import yaml
+from sphinx.application import Sphinx
+from sphinx.util import logging
+from textwrap import dedent, indent
 
-sys.path.insert(0, os.path.abspath('..'))
-sys.path.append(os.path.abspath(os.path.join(__file__, "../../src")))
+print("sys.path:", sys.path)
 
-# -- Project information -----------------------------------------------------
+import geocat.viz as gv
 
-project = 'GeoCAT-viz'
+LOGGER = logging.getLogger("conf")
 
-current_year = datetime.datetime.now().year
-copyright = u'{}, University Corporation for Atmospheric Research'.format(
-    current_year)
-author = u'GeoCAT'
+try:
+    from unittest.mock import MagicMock
+except ImportError:
+    from mock import Mock as MagicMock
+
+
+class Mock(MagicMock):
+
+    @classmethod
+    def __getattr__(cls, name):
+        return MagicMock()
+
+
+MOCK_MODULES = ["xarray", "dask", "dask.array", "dask.array.core", "cf_xarray"]
+sys.modules.update((mod_name, Mock()) for mod_name in MOCK_MODULES)
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#sys.path.insert(0, os.path.abspath('.'))
+
+# -- General configuration ------------------------------------------------
 
-# -- General configuration ---------------------------------------------------
+# If your documentation needs a minimal Sphinx version, state it here.
+#needs_sphinx = '1.0'
 
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'nbsphinx',
-    'sphinx.ext.autodoc',
     'sphinx.ext.napoleon',
+    'sphinx.ext.autodoc',
     'sphinx.ext.autosummary',
     'sphinx.ext.intersphinx',
-    'sphinx_design',
+    'sphinx.ext.mathjax',
+    'myst_nb',
+    "sphinx_design",
+    "nbsphinx",
+    'sphinx.ext.extlinks',
 ]
 
+#mathjax_path = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
+
 intersphinx_mapping = {
+    'dask': ('https://docs.dask.org/en/latest/', None),
     'python': ('http://docs.python.org/3/', None),
-    'numpy': ('http://docs.scipy.org/doc/numpy/', None),
+    'numpy': ("https://numpy.org/doc/stable", None),
     'scipy': ('https://docs.scipy.org/doc/scipy/reference/', None),
     'xarray': ('http://xarray.pydata.org/en/stable/', None),
-    'matplotlib': ('https://matplotlib.org/stable/', None),
-    'cartopy': ('https://scitools.org.uk/cartopy/docs/latest/', None)
+    'pint': ('https://pint.readthedocs.io/en/stable/', None),
+    'cftime': ('https://unidata.github.io/cftime/', None),
```

Comment:
Pretty sure we need to leave in matplotlib and cartopy here.  It looks like pint is useful, but not sure if we mention cftime at all.

Reply:
Done